### ISSUE TITLE
Fix Vuetify unit test warnings

### ISSUE THREAD

**Issue by kinow** [cylc/cylc-ui#486]
**Describe the bug**

Running unit tests, there are warnings for Vuetify that are not too hard to fix. I fixed one in `tree.vue.spec.js` for #483 .

First we need to avoid multiple Vue version warning: https://github.com/vuejs/vue-test-utils/issues/1138

We can do so by fixing our import. Importing the plug-in, it will instantiate Vue and tell it to use Vuetify. Instead, we can import Vuetify class.

Fixing this one, might bring another error, due to some missing `lang` attribute in some components. That can then be fixed with the mock from this issue: https://github.com/vuetifyjs/vuetify/issues/9923

Spent some time today troubleshooting it in order to write the tests for the Tree filters, but now that we have a working solution, we just need to apply to other tests later when we have some spare time in-between tickets.

**Release version(s) and/or repository branch(es) affected?**

**Steps to reproduce the bug**

Run unit tests.

**Expected behavior**

No Vuetify warnings in the tests.

**Screenshots**

**Additional context**

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by mirikerstein** [comment]
Hello, I'm having this issue now. Can you please give more direction about the multiple instances of vue? Thank you!

**Comment by kinow** [comment]
Hi @mirikerstein 

In my case, I'm loading this file in the tests: https://github.com/cylc/cylc-ui/blob/292d44e28de65eb870ec6860078172cb59487954/src/plugins/vuetify.js#L25

Instead, to fix the issue, I can just use the `createLocalVue` from `vue-test-utils`, and use that instance with Vuetify. That way the issue is gone. I've fixed one of the tests in this PR: https://github.com/cylc/cylc-ui/pull/483/files 

Take a look at the test `tests/unit/components/cylc/tree/tree.vue.spec.js`, and let me know if that works for you too :-) I've got to finish that PR, then will prepare another PR to fix this issue (i.e. fix all other tests with the same error).

Hope that helps
Bruno

**Comment by kinow** [comment]
Another thing that I noticed, @mirikerstein , is that calling `localVue.use(vuetify)` immediately caused the issue too. So instead I simply provided a mock.

```
const wrapper = mount(Toolbar, {
      localVue,
      vuetify,
      store,
      mocks: {
        $route,
        $vuetify: {
          application: {
            register: () => {}
          }
        },
        $t: () => {} // vue-i18n,
      }
    })
```

**Comment by kinow** [comment]
Just tested with `master`, unit test logs don't include any Vuetify warnings (or any other warnings FWIW :tada: ). @mirikerstein take a look at the linked PR's commits if you are still having a similar issue :+1: 

Closing.
